website/blog/2021-05-20-GAAD-One-Year-Later.md

```diff
@@ -0,0 +1,119 @@
+
+When we started there were 90 issues identified, since March when the gap analysis issues launched on Github:
+
+- 10 issues have been closed by the community.
+
+- 19 issues were evaluated and closed by the React Native team.
+
+- 9 pull requests were merged.
+
+- 1 pull request was merged into the React Native docs.
+
+We want to recognize and thank the React Native community for the significant progress towards a more accessible React Native over the past year. Every contributor's effort has counted in making progress on improving React Native Accessibility.
+
+<!--truncate-->
+
+## Fixes
+
+Two types of issues have been fixed in multiple components and one new functionality has been added to the API by the 9 pull requests.
+
+- An issue with Disabled state has been addressed in seven components
+
+- An issue with Selected state was addressed in two components
+
+- A new addition to the React Native API added the ability to query AccessibilityManager.getRecommendedTimeoutMillis().
+
+### Disabled State Announcement and Disable function
+
+One of the most prevalent issues found during the gap analysis was that some components do not announce or disable functionality. Now seven components announce their disabled state or disable click functionality.
+
+Announces when Disabled
+
+- `Button`
+
+- `Images`
+
+- `ImageBackground`
+
+Disables click functionality when the component has a disabled prop
+
+- `Button`
+
+- `Text`
+
+- `Pressable`
+
+- `TouchableHighlight`
+
+- `TouchableOpacity`
+
+- `TouchableNativeFeedback`
+
+- `TouchableWithoutFeedback`
+
+### Selected State Announcement
+
+There were some components that did not announce their selection when in focus. This behavior has now been fixed when the component is in focus and the AccessibilityState is set to selected or the component is changed to selected.
+
+Announces when Selected
+
+- `Button`
+
+- `TextInput`
+
+### Accessibility Timeout Setting
+
+There was previously no way to query the accessibility timeout setting on Android. The fix added the ability to query `AccessibilityManager.getRecommendedTimeoutMillis()`. This queries the "Time to take action" before the UI elements auto-dismisses or auto-progresses.
+
+## Documentation Additions
+
+The React Native documentation must be updated to reflect each addition or change to the available APIs. The [new addition to the React Native documentation](https://reactnative.dev/docs/next/accessibilityinfo#getrecommendedtimeoutmillis-android) covered the addition of `getRecommendedTimeoutMillis()` to AccessibilityInfo.
+
+## Community Involvement
+
+We want to thank all the contributors mentioned below who have submitted and merged pull requests as well as those who have reviewed and commented on issues.
+
+### Merged Pull Request
```

Comment:
Perhaps make this plural?

Reply:
Another idea: 
Maybe we should link the merged pull requests with the new functionality that we've listed above. Like: 
> Announces when Disabled
>
> - `Button` (#31001, @huzaifaaak), (...another PR, author if multiple) 
>
> - `Images` (#31252, @fabriziobertoglio1987)

Reply:
I would like to keep the original list but I can add the PR number and PR creator next to the new functionality.

Reply:
I changed it to just the PR numbers beside the components but reformated the Merged PR list to put the contributor list.